templates/usage.txt.gotmpl

```diff
@@ -1,7 +1,25 @@
 Usage: {{.ProgName}} <subcommand> [arguments]
 
+{{if .SubCommandExtendedHelp}}
+{{.SubCommandExtendedHelp}}
+{{end}}
+
 Subcommands:
 {{range .SubCommands}}
     {{.SubCommandName | printf "%-10s"}} {{.SubCommandDescription}}
 {{end}}
     version      Print version information
+
+{{if .Parameters}}
+Flags:
+{{range .Parameters}}
```

Comment:
Ensure we add `-` to the start or end of the `{{` and `}}` appropraitely to eliminate weird whitespacing @jules